Sources/PackageGraph/Pubgrub.swift

```diff
@@ -947,3 +1034,41 @@ public final class PubgrubDependencyResolver<
         }
     }
 }
+
+fileprivate func textTable(_ data: [[String]]) -> String {
```

Comment:
We can get rid of this if since it'll be difficult to format if we're reporting trace steps to a delegate